spring-cloud-kubernetes-client-config/src/main/java/org/springframework/cloud/kubernetes/client/config/KubernetesClientConfigMapPropertySource.java

```diff
@@ -42,13 +44,15 @@
 	@Deprecated
 	public KubernetesClientConfigMapPropertySource(CoreV1Api coreV1Api, String name, String namespace,
 			Environment environment) {
-		super(getName(name, namespace), getData(coreV1Api, name, namespace, environment, "", true));
+		super(getName(name, getApplicationNamespace(namespace, "Config Map", null)), getData(coreV1Api,
```

Comment:
same as for fabric8 - if someone uses this directly, they must provide the `namespace`, otherwise this will fail